scripts/build-ancillary-repository.sh

```diff
@@ -115,25 +115,10 @@ else
 fi
 echo "Running with UPSTREAM_BRANCH set to ${UPSTREAM_BRANCH}"
 
-AWS_S3_URI_VIRTUALIZATION=$(resolve_s3_uri \
-	"$AWS_S3_URI_VIRTUALIZATION" \
-	"$AWS_S3_PREFIX_VIRTUALIZATION" \
-	"dlpx-app-gate/${UPSTREAM_BRANCH}/build-package/post-push/latest")
-
-AWS_S3_URI_MASKING=$(resolve_s3_uri \
-	"$AWS_S3_URI_MASKING" \
-	"$AWS_S3_PREFIX_MASKING" \
-	"dms-core-gate/${UPSTREAM_BRANCH}/build-package/post-push/latest")
-
-AWS_S3_URI_USERLAND_PKGS=$(resolve_s3_uri \
-	"$AWS_S3_URI_USERLAND_PKGS" \
-	"$AWS_S3_PREFIX_USERLAND_PKGS" \
-	"devops-gate/master/linux-pkg-build/${UPSTREAM_BRANCH}/userland/post-push/latest")
-
-AWS_S3_URI_KERNEL_PKGS=$(resolve_s3_uri \
-	"$AWS_S3_URI_KERNEL_PKGS" \
-	"$AWS_S3_PREFIX_KERNEL_PKGS" \
-	"devops-gate/master/linux-pkg-build/${UPSTREAM_BRANCH}/kernel/post-push/latest")
+AWS_S3_URI_COMBINED_PACKAGES=$(resolve_s3_uri \
+	"$AWS_S3_URI_COMBINED_PACKAGES" \
+	"$AWS_S3_URI_COMBINED_PACKAGES" \
```

Comment:
These look the same; typo?

Reply:
It's not a typo. It's just that the way this function currently works is that it expects that the value can come from one of those 2 variables. Perhaps I could just pass `""` as 2nd argument